ern-orchestrator/src/runMiniApp.ts

```diff
@@ -238,10 +239,13 @@ export async function runMiniApp(
   function isOldAndroidRunner(): boolean {
     const paths = path.join(
       pathToRunner,
-      'app/src/main/java',
-      extra.androidConfig.packageFilePath
+      'app/src/main/java/com/walmartlabs/ern/RunnerConfig.java'
     )
-    return !fs.existsSync(paths)
+
+    if (extra && extra.forceNewAndroidRunner) {
```

Comment:
Having trouble understanding the purpose of `forceNewAndroidRunner`. Has it only been done to support tests ?

Reply:
yes, only for tests. Perhaps we can have a way to return true for this method while running tests ?

Reply:
Yes. It's bad to have code that exists only to facilitate unit tests, usually hides some kind of design flaw or just quick shortcut. Do not bother with this one, will have a look at it, will send you updated content